src/components/PeopleFilters.tsx

```diff
@@ -1,12 +1,54 @@
-export const PeopleFilters = () => {
+import { FC } from 'react';
+import classNames from 'classnames';
+import { SearchLink } from './SearchLink';
+
+type Props = {
+  handleQueryChange: (value: React.ChangeEvent<HTMLInputElement>) => void,
+  centuries: string[],
+  genders: string,
+  query: string,
+};
+
+export const PeopleFilters: FC<Props> = ({
+  handleQueryChange,
+  centuries,
+  genders,
+  query,
+}) => {
+  const filterByClick = (value: string) => {
+    if (value === 'Female') {
+      return {
+        genders: value,
+      };
+    }
+
+    if (value === 'Male') {
+      return {
+        genders: value,
+      };
+    }
+
+    return {
+      genders: null,
+    };
+  };
+
   return (
     <nav className="panel">
       <p className="panel-heading">Filters</p>
 
       <p className="panel-tabs" data-cy="SexFilter">
-        <a className="is-active" href="#/people">All</a>
-        <a className="" href="#/people?sex=m">Male</a>
-        <a className="" href="#/people?sex=f">Female</a>
+        {'All Male Female'.split(' ').map(gender => (
```

Comment:
Instead of splitting a string into an array, consider defining an array of strings directly. This makes the code more readable and maintainable.